pytorch_lightning/trainer/training_loop.py

```diff
@@ -451,7 +451,7 @@ def on_before_backward(self, batch_idx, optimizer):
 
     def _track_gradient_norm(self, batch_idx):
```

Comment:
Remove `batch_idx` as it is not used anymore?

Reply:
thanks, I missed that one

Reply:
done!